app/export.go

```diff
@@ -554,7 +632,14 @@ func (a *App) exportCustomEmoji(writer io.Writer, outPath, exportDir string, exp
 				}
 			} else {
 				filePath = filepath.Join("emoji", emoji.Id, "image")
-				emojiPaths = append(emojiPaths, filePath)
+				emojiDataList = append(emojiDataList, &imports.AttachmentImportData{
+					Id:        emoji.Id,
+					CreatorId: emoji.CreatorId,
+					CreateAt:  emoji.CreateAt,
+					UpdateAt:  emoji.UpdateAt,
+					Path:      &filePath,
+					Name:      emoji.Name,
```

Comment:
What's the reason we are adding fields like `Id`? They are not taken into account while importing right?

Reply:
@isacikgoz , we want to include the full record of the attachment in the export log so the user has everything they need to address or investigate the issue. This was requested/agreed upon by @nab-77 as we were working out the approach.